public/dist/js/buttonHandler.js

```diff
@@ -1,44 +1,44 @@
 // This function takes in a string and for each character
 // it will either hide or show a button; an uppercase
 // letter shows the button, and lowercase hides it.
-function toggleButtons(buttonToggles) {
-	for (var i = 0; i < buttonToggles.length; ++i) {
-		var firstChar = buttonToggles.charAt(i);
-		var button    = getButton(firstChar.toLowerCase());
```

Comment:
If we go down this road, we should transpile our ES6 to ES5 ASAP. This breaks on all relatively old browsers, e.g. IE.

Reply:
This were the rules set up by the linter. Is the var to const change what will brake it ?

Reply:
alright so for now ill roll this back